web/src/views/DandisetLandingView/DownloadDialog.vue

```diff
@@ -0,0 +1,177 @@
+              target="_blank"
+              rel="noopener"
+              icon
+            >
+              <v-icon
+                color="primary"
+                v-on="on"
+              >
+                mdi-help-circle
+              </v-icon>
+            </v-btn>
+          </template>
+          More help on download
+        </v-tooltip>
+      </v-card-title>
+      <v-list class="pa-0">
+        <v-list-item dense>
+          Use this command in your DANDI CLI
+        </v-list-item>
+        <v-list-item dense>
+          <CopyText
+            :text="defaultDownloadText"
+            icon-hover-text="Copy command to clipboard"
+            dense
+            filled
+            outlined
+          />
+        </v-list-item>
+        <v-expansion-panels>
+          <v-expansion-panel v-if="availableVersions.length > 0">
+            <v-expansion-panel-header>
+              Download a different version
+            </v-expansion-panel-header>
+            <v-expansion-panel-content>
+              <v-list class="pa-0">
+                <v-list-item dense>
+                  <v-radio-group v-model="selectedDownloadOption">
+                    <v-radio
+                      label="Draft"
+                      value="draft"
+                    />
+                    <v-radio
+                      label="Latest version"
+                      value="latest"
+                    />
+                    <v-radio
+                      label="Other version"
+                      value="other"
+                    />
+                    <v-select
+                      v-if="selectedDownloadOption == 'other'"
+                      v-model="selectedVersion"
+                      :items="availableVersions"
+                      item-text="version"
+                      item-value="index"
+                      dense
+                    />
+                  </v-radio-group>
+                </v-list-item>
+                <v-list-item dense>
+                  <CopyText
+                    :text="customDownloadText"
+                    icon-hover-text="Copy command to clipboard"
+                    color="primary"
+                    dense
+                    outlined
+                    filled
+                  />
+                </v-list-item>
+              </v-list>
+            </v-expansion-panel-content>
+          </v-expansion-panel>
+          <v-expansion-panel>
+            <v-expansion-panel-header>
+              Don't have DANDI CLI?
+            </v-expansion-panel-header>
+            <v-expansion-panel-content>
+              <v-list>
+                <v-list-item>
+                  <div>
+                    Install the Python client (DANDI CLI)
+                    in a Python 3.6+ environment using command:
+                    <kbd>pip install dandi</kbd>
+                  </div>
+                </v-list-item>
+                <ApiKeyItem />
```

Comment:
I think we need to have some note here if you're not logged in. Currently, it will just display that text with an empty api key section. I think we some extra information here that informs the user that they need to create an account in order to use the DANDI CLI, and maybe a tool-tip over the `ApiKeyItem` that tells them that they need to be logged in to retrieve their API key.

Reply:
one of the requirements in #473 is that the reference to the API key be removed. users don't need to create an account to use the dandi cli for downloading.

Reply:
Yes, we don't want the API key component in download.